src/tests/unit/tests/electron/platform/android/device-config-fetcher.test.ts

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { AxiosResponse } from 'axios';
+import { DeviceConfig, DeviceConfigParser } from 'electron/platform/android/device-config';
+import {
+    createDeviceConfigFetcher,
+    DEVICE_CONFIG_FETCH_TIMEOUT_MS,
+    DeviceConfigFetcher,
+} from 'electron/platform/android/device-config-fetcher';
+import { HttpGet } from 'electron/platform/android/fetch-scan-results';
+import { IMock, It, Mock, MockBehavior, Times } from 'typemoq';
+
+describe('createDeviceConfigFetcher', () => {
+    const successfulResponse = {
+        status: 200,
+        data: { opaqueProperty: 'any value' },
+    } as AxiosResponse<any>;
+    const errorResponse = {
+        status: 500,
+        statusText: 'internal server error',
+    } as AxiosResponse<any>;
+
+    let mockHttpGet: IMock<HttpGet>;
+    let mockParser: IMock<DeviceConfigParser>;
+    let testSubject: DeviceConfigFetcher;
+
+    beforeEach(() => {
+        mockHttpGet = Mock.ofType<HttpGet>(undefined, MockBehavior.Strict);
+        mockParser = Mock.ofType<DeviceConfigParser>(undefined, MockBehavior.Strict);
+        testSubject = createDeviceConfigFetcher(mockHttpGet.object, mockParser.object);
+    });
+
+    function verifyAllMocks(): void {
+        mockHttpGet.verifyAll();
+        mockParser.verifyAll();
+    }
+
+    it('coordinates HttpGet and DeviceConfigParser in the happy path', async () => {
+        const port = 123;
+        const expectedUrl = `http://localhost:123/AccessibilityInsights/config`;
+        const expectedRequestConfig = { timeout: DEVICE_CONFIG_FETCH_TIMEOUT_MS };
+        const parserOutput: DeviceConfig = { deviceName: 'device', appIdentifier: 'app' };
+
+        mockHttpGet
+            .setup(m => m(expectedUrl, expectedRequestConfig))
+            .returns(() => Promise.resolve(successfulResponse))
+            .verifiable(Times.once());
```

Comment:
are we able to remove verifiable here? I'm looking at https://github.com/florinn/typemoq#verify-all-expectations-at-once